assets/javascripts/discourse/controllers/admin-plugins-discourse-subscriptions-products-show-plans-show.js.es6

```diff
@@ -10,6 +10,26 @@ export default Controller.extend({
   selectedCurrency: Ember.computed.alias("model.plan.currency"),
   selectedInterval: Ember.computed.alias("model.plan.interval"),
 
+  @discourseComputed("model.plan.metadata.group_name")
+  selectedGroup(groupName) {
+    if (!groupName) {
```

Comment:
Mostly so you can tell I actually review this... you think you could do `return groupName || "no-group"`

Reply:
Also I just saw above there are some `Ember` global variable uses. Might want to start importing those as you work on the plugin. Or do a sweep to import stuff like `Ember.computed.alias`

Reply:
Should be fixed here. I'll open a 2nd PR with the imports.